### PR TITLE
feat: add python playwright image

### PR DESCRIPTION
**Pull request by drobnikj**
Ultimately, I used the official playwright image and ipyenv to install specific python into the image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,20 @@ You can then run the workflow by providing the following inputs:
...
 - A version of the `puppeteer` package that should be pre-installed in the images that use Puppeteer.
+
+### Adding a new actor image
+
+To create a new image, which is not yet published in Apify DockerHub organization.
+You need access to the organization and rights to create a new repository.
+After, you need to follow these steps:
+
+1. Create a new folder with the same name as the package you want to create without the prefix `actor-`. 
+For image `apify/actor-node`, create folder `name`.
```
Comment (fnesveda) [inline README.md:35]:
```suggestion
For image `apify/actor-node`, create folder `node`.
```

File: .github/workflows/release-python-playwright.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        run: docker run ${{ fromJson(steps.prepare-tags.outputs.result).firstImageName }}
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Push images
+        run: docker push apify/actor-${{ matrix.image-name }} --all-tags
```
Comment (fnesveda) [inline .github/workflows/release-python-playwright.yml:100]: You should run these steps only when the event is not `pull_request`, same as it is in the Node workflows

**Review by fnesveda — CHANGES_REQUESTED** [review]
Looks great! Just one comment about the publishing workflow

**Review by jirimoravcik — CHANGES_REQUESTED** [review]

File: .github/workflows/release-python-playwright.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+name: Release python playwright images
```
Comment (jirimoravcik) [inline .github/workflows/release-python-playwright.yml:1]: rename this to `.yaml`

File: python-playwright/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+# Preinstall the latest versions of setuptools and wheel for faster package installs
+RUN pip install --upgrade setuptools wheel
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install the specified Apify SDK version and Python Playwright version
+ARG PLAYWRIGHT_VERSION=1.30.0
```
[the remark this answers is not in the document]
Comment (drobnikj) [reply python-playwright/Dockerfile]: you need to have AVG before the docker step you want to use it, but I put the default version into env.

[the remark this answers is not in the document]
Comment (drobnikj) [reply python-playwright/Dockerfile:51]: It is by design https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact, you need to define it as the default  value it not there after FROM statement.

**Review by jirimoravcik — APPROVED** [review]

**Review by fnesveda — APPROVED** [review]

**Pull request merged**